src/feature/dirauth/process_descs.c

```diff
@@ -47,10 +47,9 @@ static void directory_remove_invalid(void);
 static was_router_added_t dirserv_add_extrainfo(extrainfo_t *ei,
                                                 const char **msg);
 static uint32_t
-dirserv_get_status_impl(const char *fp, const char *nickname,
-                        uint32_t addr, uint16_t or_port,
-                        const char *platform, const char **msg,
-                        int severity);
+dirserv_get_status_impl(const char *fp, const uint8_t *ed25519_key,
```

Comment:
The new argument should be "const ed25519_public_key_t *ed25519_public_key".

Reply:
46d7d140e2f4927059f82525e265a7158e6314b5